praw/models/reddit/poll.py

```diff
@@ -107,13 +105,14 @@ def __setattr__(self, attribute: str, value: Any):
     def option(self, option_id: str) -> PollOption:
         """Get the option with the specified ID.
 
-        :param option_id: The ID of a poll option, as a ``str``.
-        :returns: The specified :class:`.PollOption`.
+        :param option_id: The ID of a poll option, as a ``str``. :returns: The specified
```

Comment:
This `:returns` should be on its own line. What let to it being pulled up here? I'm skimming the PR, so it makes me slightly nervous that something else like this may have happened (though honestly stakes are low — we can always correct documentation mistakes when we notice them, so it's not a big deal).

Reply:
> This `:returns` should be on its own line. What let to it being pulled up here? I'm skimming the PR, so it makes me slightly nervous that something else like this may have happened (though honestly stakes are low — we can always correct documentation mistakes when we notice them, so it's not a big deal).

This was an uncaught mistake. I can do a search to find these. Also, I can go through all the changes again later today.